mslice/models/cmap.py

```diff
@@ -3,6 +3,7 @@
 """
 from mslice.util import MPL_COMPAT
 
+DEFAULT_CMAP = 'viridis'
```

Comment:
:+1:

The `allowed_cmaps` function below removes viridis if it is not available (some old Macs still don't have it!). Maybe we rewrite this as:

```python
ALLOWED_CMAPS = ['jet', 'summer', 'winter', 'coolwarm']
if not MPL_COMPAT:
    ALLOWED_CMAPS.insert(0, 'viridis')

DEFAULT_CMAP = ALLOWED_CMAPS[0]
```

and get rid of the function?